src/makeRobotAccountant.js

```diff
@@ -5,8 +5,20 @@
  * @return {function}
  */
 
-function makeRobotAccountant() {
-  // write code here
+function makeRobotAccountant(a) {
+  let count = 0;
+
+  return (b) => {
+    return (c) => {
```

Comment:
Better rename to `firstNumber` and `secondNumber` for example instead of `b and c`.